src/modules/reward/hooks/queries/useRewardConstants.js

```diff
@@ -0,0 +1,30 @@
+import { REWARD_CONSTANTS } from 'src/const/queries';
+import {
+  API_VERSION,
+} from 'src/const/config';
+import { useCustomQuery } from '@common/hooks';
+
+/**
+ * Creates a custom hook to fetch reward constants
+ *
+ * @param {object} [configuration] - the custom query configuration object
+ * @param {object} [configuration.config] - the query config
+ * @param {string} [configuration.options] - the query options
+ *
+ * @returns the query object
+ */
+
+export const useRewardConstants = ({ config: customConfig = {}, options } = { }) => {
+  const config = {
+    url: `/api/${API_VERSION}/reward/constants`,
+    method: 'get',
+    event: 'get.reward.constants',
+    ...customConfig,
+  };
+
+  return useCustomQuery({
+    keys: [REWARD_CONSTANTS],
+    config,
+    options,
+  });
+};
```

Comment:
because this would be a hook to get rewards constants from service, @soroushm @oskarleonard I think we had a discussion during grooming that the query hook should have a large stale/cache time, would this still be the case ?

Reply:
Hmm, that makes sense. Maybe we should set it to `Infinity` if those values almost never change. Do you remember how often these values change?